### PR TITLE
Speak to when a DID and DID Document start existing.

### PR DESCRIPTION
**Pull request by msporny**
This PR has been raised to address #729. It elaborates on when a DID and DID Document begin to exist. /cc @Kalanamith

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/765.html" title="Last updated on Jun 27, 2021, 6:30 PM UTC (2f5075f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/765/443e944...2f5075f.html" title="Last updated on Jun 27, 2021, 6:30 PM UTC (2f5075f)">Diff</a>

### CODE REVIEW

**Review by kdenhartog — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -818,7 +818,9 @@ <h2>Identifier</h2>
 This section describes the formal syntax for <a>DIDs</a> and <a>DID URLs</a>.
 The term "generic" is used to differentiate the syntax defined here from syntax
 defined by <em>specific</em> <a>DID methods</a> in their respective
-specifications.
+specifications. How and when these identifiers are created are described in
+<a href="#method-operations"></a> and
+<a href="#creation-of-a-did"></a>.
```
Comment (TallTed) [inline index.html:823]:
```suggestion
specifications. The creation processes, and their timing, for <a>DIDs</a> and
<a>DID URLs</a> are described in <a href="#method-operations"></a> and
<a href="#creation-of-a-did"></a>.
```

File: index.html [hunk 2]
```diff
@@ -5747,6 +5749,22 @@ <h2>application/did+ld+json</h2>
 
   <section class="appendix">
     <h1>Architectural Considerations</h1>
+
+    <section>
+      <h2>Creation of a DID</h2>
+      <p>
+The creation of a <a>DID</a> is a process that is defined by each <a>DID
+Method</a>. Some <a>DID Methods</a>, such as <code>did:key</code>, are purely
+generative such that a <a>DID</a> and a <a>DID Document</a> is generated by
```
Comment (TallTed) [inline index.html:5758]:
```suggestion
generative, such that a <a>DID</a> and a <a>DID Document</a> are generated by
```

File: index.html [hunk 3]
```diff
@@ -5747,6 +5749,22 @@ <h2>application/did+ld+json</h2>
...
+transforming a single piece of cryptographic material into a conformant
+<a>representation</a>. Other <a>DID Methods</a> require the use of a
```
Comment (TallTed) [inline index.html:5760]:
```suggestion
<a>representation</a>. Other <a>DID Methods</a> might require the use of a
```

File: index.html [hunk 4]
```diff
@@ -5747,6 +5749,22 @@ <h2>application/did+ld+json</h2>
...
+<a>verifiable data registry</a> where the <a>DID</a> and <a>DID Document</a>
```
Comment (TallTed) [inline index.html:5761]:
```suggestion
<a>verifiable data registry</a>, where the <a>DID</a> and <a>DID Document</a>
```

File: index.html [hunk 5]
```diff
@@ -5747,6 +5749,22 @@ <h2>application/did+ld+json</h2>
...
+are recognized to exist by third parties only when the registration has been
+completed, as defined by the respective <a>DID Method</a>.
```
Comment (TallTed) [inline index.html:5763]:
```suggestion
completed, as defined by the respective <a>DID Method</a>. Other processes
may be defined by the respective <a>DID Method</a>.
```

**Review by TallTed — CHANGES_REQUESTED** [review]
I think these tweaks will increase clarity without changing meaning.

File: index.html (hunk 2 above)
Comment (rhiaro) [inline index.html:5758]:
```suggestion
method</a>. Some <a>DID methods</a>, such as <code>did:key</code>, are purely
generative such that a <a>DID</a> and a <a>DID document</a> is generated by
```

File: index.html (hunk 3 above)
Comment (rhiaro) [inline index.html:5760]:
```suggestion
<a>representation</a>. Other <a>DID methods</a> require the use of a
```

File: index.html (hunk 4 above)
Comment (rhiaro) [inline index.html:5761]:
```suggestion
<a>verifiable data registry</a> where the <a>DID</a> and <a>DID document</a>
```

File: index.html (hunk 5 above)
Comment (rhiaro) [inline index.html:5763]:
```suggestion
completed, as defined by the respective <a>DID method</a>.
```

**Review by rhiaro — CHANGES_REQUESTED** [review]
and +1 to @TallTed's suggested changes

**Comment by TallTed** [comment]
I don't think github will do the desired thing where our suggestions overlap, so I've blended @rhiaro's changes into mine, where they fit.  There will remain one or two upper-to-lower case letter shifts after applying all of mine, but applying @rhiaro's would then revert some of mine, unless those also get tweaked to include my changes.

Alternatively, it might be best to manually apply all the suggestions we both made.

**Review by brentzundel — APPROVED** [review]

**Review by peacekeeper — APPROVED** [review]
Agree with this PR and with tweaks by @TallTed and @rhiaro

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**